contracts/CourtPresaleActivate.sol

```diff
@@ -184,28 +184,29 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
         uint256 contributionTokenAmount = uniswapExchange.ethToTokenSwapInput.value(msg.value)(_minTokens, _deadline);
 
         // buy in presale
-        _buyAndRegisterAsJuror(msg.sender, contributionTokenAmount, contributionToken, _activate);
+        _buyAndActivateAsJuror(msg.sender, contributionTokenAmount, contributionToken, _activate);
     }
 
     function _hasData(bytes memory _data) internal pure returns (bool) {
```

Comment:
Small nitpick, we normally move the `pure` functions to be after non-view and state changing functions (i.e. this should come after `_buyAndActivateAsJuror()`).

Reply:
Yes, too bad I added the solium config files later in another PR

Reply:
(Update: I added solium stuff and it didn't complain... :thinking: )